### ISSUE TITLE
Google Maps don't find location / "not a LatLngBounds ..."

### ISSUE THREAD

**Issue by georgjaehnig** [ProfessionalWiki/Maps#697]
### Setup

- Maps version: 9.0.7
- MW version: 1.37.2
- PHP version: 8.0.19
- SMW version (if applicable): 

#### Detailed description of the issue and a [stack trace](https://www.semantic-mediawiki.org/wiki/Help:Identifying_bugs) if applicable:

It seems the Google maps don't find the location anymore, even on the [example page](https://maps.extension.wiki/wiki/Displaying_Google_maps). The console reports a JS error:

    not a LatLngBounds or LatLngBoundsLiteral: not an Object

Can someone else confirm?

#### Steps to reproduce the observation (recommendation is to use the [sandbox](https://sandbox.semantic-mediawiki.org)):

Go on the [example page](https://maps.extension.wiki/wiki/Displaying_Google_maps).

**Comment by JeroenDeDauw** [comment]
Looks like a general geocoding issue. Can you try switching to leaflet and see if it works there? I expect it does not.

**Comment by georgjaehnig** [comment]
It's not working on the [very example page of this extension (maps.extension.wiki)](https://maps.extension.wiki/wiki/Displaying_Google_maps).  So I can't make the changes for testing myself.

Interestingly, a few [Google maps do work](https://maps.extension.wiki/wiki/Displaying_Google_maps#Customization).

Looking [at leaflet](https://maps.extension.wiki/wiki/Displaying_Leaflet_maps), a few work and a few don't work, too.

**Comment by georgjaehnig** [comment]
It might have to do something with **"upgrading the Google Cloud Platform"**. 

I've just did that for my own wiki: [went here](https://console.cloud.google.com/project/_/billing/enable) and got it offered.

And now, it works for all *new* Geocoding queries I am sending!

So if before I had

    {{#display_map:Brandenburg Gate Berlin | service=google}}

it still doesn't work, but if I replace it with a new query like

    {{#display_map:TV tower Berlin | service=google}}

the map displays fine.

So I guess it's now a caching issue and will resolve itself.

**Comment by JeroenDeDauw** [comment]
Did you set `egMapsDefaultGeoService` in your configuration?

**Comment by georgjaehnig** [comment]
Yes:

    $egMapsDefaultGeoService = 'google';

**Comment by JeroenDeDauw** [comment]
Did you already have that when you reported the issue?

**Comment by georgjaehnig** [comment]
Yes. But again: I reported what I [have seen on maps.extension.wiki](https://maps.extension.wiki/wiki/Displaying_Google_maps), not what happens on my wiki!

Do you also see the maps not being displayed on maps.extension.wiki?

**Comment by JeroenDeDauw** [comment]
Yes, there appears to be a geocoding issue on maps.extension.wiki. Maps with coordinates show, those with addresses do not.

**Comment by georgjaehnig** [comment]
Yup, and when you try to geocode a completely new place, e.g.

    {{#display_map:TV tower Berlin | service=google}}

does it get displayed?

**Comment by georgjaehnig** [comment]
So on my wiki, it [works again](https://expatwiki.org/Germany/Shopping/Sunday/Berlin). I conclude the problem was to

- some changes / expired trial on Google Cloud Platform that I had to upgrade
- after the upgrade, Google Geocoding API still had the old (faulty) queries in cache. (Adding maps with new queries already worked.) Now, after 3 hours, the cache got cleared
